content/recipes/prisma.md

```diff
@@ -181,6 +181,11 @@ With your Prisma models in place, you can generate your SQL migration files and
 ```bash
 $ npx prisma migrate dev --name init
 ```
+OR for prisma using mongodb
+
+```bash
+$ prisma db push
```

Comment:
npx?

Reply:
When user have prismaCLI instaled can use directly command prisma {command} {argument}

Reply:
I know but in the other code snippets (in this chapter) we use npx

Reply:
Ok, i'll revise that and submit again. Thanks !